.babelrc

```diff
@@ -1,18 +1,11 @@
 {
-  "presets": [["env", { "modules": false }], "stage-2"],
+  "presets": ["@babel/env"],
+  "plugins": [
+    ["@babel/proposal-object-rest-spread", { "loose": true }, "unique"]
```

Comment:
What does `"unique"` do here? I can't find anything about it in the babel docs

Reply:
Ok, I found [this](https://babeljs.io/docs/en/options#pluginentry-presetentry), but that name link leads nowhere :/ So still no clue

Reply:
Parcel was complaining about duplicate plugins, this fixed it